rmgpy/rmg/input.py

```diff
@@ -97,45 +98,65 @@ def database(
+        raise InputError("In catalyst_properties section you should only specify a 'metal' shortcut " 
+                         "or the surfaceSiteDensity and bindingEnergies, but not both.")
 
+    if metal:
+        try:
+            logging.info("Using catalyst surface properties from metal %r.", metal)
+            rmg.binding_energies = metal_db.get_binding_energies(metal)
+            rmg.surface_site_density = metal_db.get_surface_site_density(metal)
+        except DatabaseError:
+            logging.error('Metal %r missing from surface library', metal)
+            raise
+    else: # metal not specified
+        if bindingEnergies is None:
+            rmg.binding_energies = metal_db.get_binding_energies("Pt111")
+            logging.info("Using default binding energies, Pt(111)")
+        else:
+            rmg.binding_energies = convert_binding_energies(bindingEnergies)
+    
 
-def convert_binding_energies(bindingEnergies):
+        if surfaceSiteDensity is None:
+            rmg.surface_site_density = metal_db.get_surface_site_density("Pt111")
+            logging.info("Using default surface site density, Pt(111)")
+        else:
+            rmg.surface_site_density = SurfaceConcentration(*surfaceSiteDensity)
+
+    logging.info("Using binding energies:\n%r", rmg.binding_energies)
+    logging.info("Using surface site density: %r", rmg.surface_site_density)
+
+
+def convert_binding_energies(binding_energies):
     """
-    Process the bindingEnergies from the input file.
-    If "None" is passed, then it returns Pt(111) values.
+    Process binding_energies dictionary from the input file
+    
+    It converts the values into Energy quantities, and checks that 
+    all elements C,H,O, and N are present.
 
-    :param bindingEnergies: a dictionary of element symbol: binding energy pairs (or None)
+    :param binding_energies: a dictionary of element symbol: binding energy pairs
     :return: the processed and checked dictionary
     """
-    if bindingEnergies is None:
-        bindingEnergies = {  # default values for Pt(111)
-            'C': (-6.750, 'eV/molecule'),
-            'H': (-2.479, 'eV/molecule'),
-            'O': (-3.586, 'eV/molecule'),
-            'N': (-4.352, 'eV/molecule'),
-        }
-        logging.info("Using default binding energies for Pt(111):\n{0!r}".format(bindingEnergies))
-    if not isinstance(bindingEnergies, dict):
+    if not isinstance(binding_energies, dict):
         raise InputError("bindingEnergies should be None (for default) or a dict.")
     new_dict = {}
     for element in 'CHON':
         try:
-            new_dict[element] = Energy(bindingEnergies[element])
+            new_dict[element] = Energy(binding_energies[element])
         except KeyError:
-            logging.error('Element {} missing from bindingEnergies dictionary'.format(element))
+            logging.error('Element {} missing from binding energies dictionary'.format(element))
             raise
```

Comment:
While I think this is a good idea, I think it should be included in the next PR as it isn't directly related to metal attributes

Reply:
I've actually put a lot of thought into this and I came to the conclusion that it makes more sense to always load all 4 binding energies to make sure _all_ training reactions can be properly loaded in (especially because we have such sparse data)!
But why would you want to load reactions that have N in them if the model won't have any N?
The training reactions may have N in them and you may want to use that node/average of nodes for estimating rate rules.